Fix for the following link error in 2.50.2:

Undefined                       first referenced
 symbol                             in file
_ZN7WebCore4toJSEPN3JSC14JSGlobalObjectEPNS_17JSDOMGlobalObjectERNS_15BasicCredentialE Source/WebCore/CMakeFiles/WebCore.dir/./__/__/WebCore/DerivedSources/unified-sources/UnifiedSource-f8afad56-13.cpp.o
ld: fatal: symbol referencing errors
clang++: error: linker command failed with exit code 1 (use -v to see invocation)

See https://github.com/WebKit/WebKit/commit/730bffd856d2a1e56dd3bd2a0702282f19c5242a


--- webkitgtk-2.50.2/Source/WebCore/Modules/credentialmanagement/BasicCredential.idl~	2025-11-11 19:11:08.870967100 +0000
+++ webkitgtk-2.50.2/Source/WebCore/Modules/credentialmanagement/BasicCredential.idl	2025-11-21 16:56:14.001813938 +0000
@@ -25,7 +25,6 @@
 
 [
     InterfaceName=Credential,
-    CustomToJSObject,
     EnabledByQuirk=shouldExposeCredentialsContainer,
     Exposed=Window,
     SecureContext
